tests/HTTP/MessageTest.php

```diff
@@ -278,4 +278,8 @@ private function createCallback($content)
 
 class MessageMock extends Message
 {
+    public function __toString(): string
+    {
+        return 'mock text';
+    }
```

Comment:
This needs `__toString` because Message implements MessageInterface and MessageInterface now has __toString